src/azul/service/manifest_controller.py

```diff
@@ -102,17 +106,17 @@ def get_manifest_async(self,
                 'Retry-After': token.wait_time
             }
         else:
+            manifest_url_func = self.manifest_url_func if fetch else None
+            command_line = self.service.command_line(format_,
+                                                     catalog,
+                                                     filters,
+                                                     manifest,
+                                                     manifest_url_func)
             body = {
                 'Status': 302,
-                'Location': manifest.location
+                'Location': manifest.location,
+                **command_line
             }
-            try:
-                command_line = manifest.properties['command_line']
-            except KeyError:
-                pass
-            else:
-                body['CommandLine'] = command_line
-
```

Comment:
> What was the reason for removing the command_line nested key here and re-implementing the same nesting elsewhere? The original structure feels better organized.

In a post standup discussion Hannes suggested this excerpt was a bad smell. I believe it has to do with the field being response specific and having to be translated between camel case and snake case. The nesting was removed from the ManifestGenerator and implemented at the ManifestService level a bit higher in the system logic. That was my attempt at addressing Hannes' observation, if further deliberation is needed we should jump on Zoom. :)